_common

```diff
@@ -255,3 +255,7 @@ find_latest_published_tumbleweed_image() {
     echo "$image"
 }
 
+fetch-vars-json() {
+    local tid=$1
+    runcurl "${args[@]}" -sS "$host_url/tests/$tid"/file/vars.json || return $?
```

Comment:
`args` won't contain anything useful here:
```suggestion
    runcurl "${curl_args[@]}" -sS "$host_url/tests/$tid"/file/vars.json || return $?
```
If shellcheck complains, then define it at the top in the same way as client_args:
```
curl_args=()
```